File: Leetcode Challenge/February/Python/Majority Element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        
        e=0
        cnt=0
        
        
        for i in range(len(nums)):
            if cnt==0:
                e=nums[i]
            
            if e==nums[i]:
                cnt+=1
            else:
                cnt-=1
        
        return e

File: Leetcode Challenge/February/java/Majority Element.java
class Solution {
    public int majorityElement(int[] nums) {
        int count=0, res = 0;
        for (int num: nums) {
            if (count==0)
                res = num;
            if (num!=res)
                count--;
            else
                count++;
        }
        return res;
    }

}
